src/mappings.go

```diff
@@ -176,6 +139,19 @@ func convertMappingsJsonToMappings(json MappingsJson) (Mappings, error) {
 	return m, nil
 }
 
+func mergeMappingsFromDefault(dist *Mappings, platform string) error {
```

Comment:
OK but what about https://github.com/lambdalisue/dotfiles/blob/support-mappings-for-linux-and-darwin/src/mappings.go#L155 ? Shall I fix this as well?